packages/odyssey-react-mui/src/TextField.tsx

```diff
@@ -37,7 +38,7 @@ export type TextFieldProps = {
   /**
    * End `InputAdornment` for this component.
    */
-  endAdornment?: ReactNode;
+  endAdornment?: string | ReactElement<typeof Icon>;
```

Comment:
I was thinking of making this a new type, but then it shouldn't show up as easily when hovering. This might change in the future, but its 2 places, so I figured it'd be safe.